apps/prairielearn/package.json

```diff
@@ -85,14 +85,14 @@
     "bootstrap-table": "1.21.2",
     "byline": "^5.0.0",
     "chalk": "^5.4.1",
-    "cheerio": "^1.1.0",
+    "cheerio": "^1.1.2",
     "clipboard": "^2.0.11",
     "clsx": "^2.1.1",
     "cookie-parser": "^1.4.7",
-    "cropperjs": "^2.0.0",
+    "cropperjs": "^2.0.1",
```

Comment:
@miguelaenlle this picked up your change! We could back out the patch in our asset handling after this if desired. Or, we could just leave it in forever 🤷